modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -59,9 +68,9 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
                               ptcpnt_id: target_veteran.participant_id.present?,
                               birls_id: target_veteran.birls_id.present?,
                               icn: target_veteran&.mpi_icn)
-      rescue ::Common::Exceptions::UnprocessableEntity, MPI::Errors::ArgumentError
+      rescue MPI::Errors::ArgumentError
```

Comment:
is this update from the MPI rework? Assuming we don't have the potential of the `UnprocessableEntity` anymore?